libs/gi/page-archive/src/index.tsx

```diff
@@ -94,6 +94,7 @@ export default function PageArchive() {
         fallback={<Skeleton variant="rectangular" width="100%" height={1000} />}
       >
         <Routes>
+          <Route index element={<TabArtifact />} />
           <Route path="artifacts" element={<TabArtifact />} />
           <Route path="weapons" element={<TabWeapon />} />
           <Route path="characters" element={<TabCharacter />} />
```

Comment:
I think we should keep the `/archive/artifacts` link, instead of using the index for artifacts. This solution likely works better.
```suggestion
          <Route path="characters" element={<TabCharacter />} />
          <Route path="*" element={<Navigate to="artifacts" replace />} />
```